tasks/main.yml

```diff
@@ -46,28 +46,34 @@
         | map('basename') | map('splitext') | map('first') | unique | list }}
     cacheable: true
 
-- name: Find policy module files
+- name: Find subpolicy files
   find:
     paths:
       - /usr/share/crypto-policies/policies/modules/
       - /etc/crypto-policies/policies/modules/
     patterns: '*.pmod'
-  register: __crypto_policies_policy_module
+  register: __crypto_policies_subpolicies
 
-- name: Set available policy modules fact
+- name: Set available subpolicies fact
   set_fact:
-    crypto_policies_available_modules: >-
-      {{ __crypto_policies_policy_module.files | map(attribute='path')
+    crypto_policies_available_subpolicies: >-
+      {{ __crypto_policies_subpolicies.files | map(attribute='path')
         | map('basename') | map('splitext') | map('first') | unique | list }}
     cacheable: true
 
+- name: Set crypto_policies_available_modules fact (deprecated)
```

Comment:
No need to create another task with the `set_fact` module, you can set several tasks within one `set_fact- module.

Reply:
Given that this is a deprecated fact, I would be fine with keeping it separate so it is clearly separated from the current facts.

Reply:
@spetrosi, if you insist on merging these, you'll also have to show me how to refer to avoid code duplication; I couldn't quite figure it out.

Reply:
I am fine with keeping the deprecated fact separated